src/pages/TransactionPage.tsx

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Transaction from '../components/Transaction';
-import { Address } from '@solana/web3.js';
+import { Address, Signature, signature } from '@solana/web3.js';
+import { _fetch } from '../fetch';
 
-// Below address is based on local machine. Need to update it dynamically.
 const LOCAL_WALLET_ADDRESS = 
-    'C1jTLcGBRcKm5XdH34ES9aL7PyJJKqXMeEjy46CJgoXp' as Address<'C1jTLcGBRcKm5XdH34ES9aL7PyJJKqXMeEjy46CJgoXp'>;  
+    '91RzsA7P8hYDqswUqapHWLSjGiMCFdT3kkoGkcpWZerW' as Address<'91RzsA7P8hYDqswUqapHWLSjGiMCFdT3kkoGkcpWZerW'>;  
 
 export default function TransactionPage() {
+    const [data, setData] = useState<Array<Signature>>(['fdasfsf' as Signature]);
+
+    useEffect(() => {
+        const fetchSignaturesForAddress = async () => {
+            const body = { address: LOCAL_WALLET_ADDRESS };
+            let signatures = await _fetch('/api/getSignaturesForAddress',body) as Array<Signature>;
+
+            // Display on UI that no transactions are found for this address
+            if(signatures){
+                setData(signatures)
+            }
+        };
+        fetchSignaturesForAddress();
+    },[]);
+
+    if(data != undefined){
+        return (
+            <>
+                <h1>This is transaction page</h1>
+                {data.map(signature => <Transaction signature={signature}/>)}
+            </>
+        )
+    }
```

Comment:
The page itself should be static, and the content within can be dynamic.

```suggestion
  return (
    <>
      <h1> This is transaction page </h1>
      { data
        ? {data.map(signature => <Transaction signature={signature}/>}
        : <h1> Loading transactions... </h1>
      }
    </>
  )
```